test/commands/new_test.rb

```diff
@@ -825,9 +826,844 @@ def container_options
+        content = @root.join('apps/controllers/home/index.rb').read
+        content.must_match %(module Chirp::Controllers::Home)
+        content.must_match %(class Index)
+        content.must_match %(include Chirp::Action)
+        content.must_match "def call(params)"
+      end
+    end
+
+    describe 'apps/views/application_layout.rb' do
+      it 'generates it' do
+        content = @root.join('apps/views/application_layout.rb').read
+        content.must_match %(module Chirp)
+        content.must_match %(module Views)
+        content.must_match %(class ApplicationLayout)
+        content.must_match %(include Chirp::Layout)
+      end
+    end
+
+    describe 'apps/templates/application.html.rb' do
+      it 'generates it' do
+        content = @root.join('apps/templates/application.html.erb').read
+        content.must_match %(<title>Chirp</title>)
+        content.must_match %(<%= yield %>)
+      end
+    end
+
+    describe 'apps/views/home/index.rb' do
+      it 'generates it' do
+        content = @root.join('apps/views/home/index.rb').read
+        content.must_match %(module Chirp::Views::Home)
+        content.must_match %(class Index)
+        content.must_match %(include Chirp::View)
+      end
+    end
+
+    describe 'apps/templates/home/index.html.rb' do
+      it 'generates it' do
+        content = @root.join('apps/templates/home/index.html.erb').read
+        content.must_match %(<h1>Welcome to Lotus!</h1>)
+        content.must_match %(<h3>This template is rendered by <code>Chirp::Views::Home::Index</code> and it's available at: <code>apps/templates/home/index.html.erb</code></h3>)
+      end
+    end
+
+    describe 'apps/public/javascripts' do
+      it 'generates it' do
+        @root.join('apps/public/javascripts').must_be :exist?
+      end
+    end
+
+    describe 'apps/public/stylesheets' do
+      it 'generates it' do
+        @root.join('apps/public/stylesheets').must_be :exist?
+      end
+    end
+
+    describe 'testing' do
+      describe 'when minitest (default)' do
+        describe 'spec/features' do
+          it 'generates it' do
+            @root.join('spec/features').must_be :directory?
+          end
+        end
+
+        describe 'spec/controllers' do
+          it 'generates it' do
+            @root.join('spec/controllers').must_be :directory?
+          end
+        end
+
+        describe 'spec/views' do
+          it 'generates it' do
+            @root.join('spec/views').must_be :directory?
+          end
+        end
+      end
+    end
+  end
+
+  describe 'when app_name is .' do
+    def container_options
+      Hash[architecture: architecture, application: 'web', application_base_url: '/', lotus_head: false, database: 'filesystem']
+    end
+
+    def clear_root_folder
+      FileUtils.rm_rf("#{@root}/.", secure: true)
+    end
+
+    let(:opts)      { container_options }
+    let(:app_name)  { '.' }
+
+    before do
+      clear_root_folder
+      capture_io { command.start }
```

Comment:
I still have problem with the tests here. I'm not sure why `.lotusrc` is created before the generator is called here. It makes the tests stucked waiting for user input. Could you give me some insight about this? 
Cc: @jodosha, @joneslee85